src/classes/App.js

```diff
@@ -6,6 +6,15 @@ import '../styles/App.css';
 import '../styles/human.css';
 
 class App extends Component {
+
+    componentWillMount(){
```

Comment:
This method was renamed to `UNSAFE_componentWillMount` in a prior version of react, and will be totally removed in 17. Can we use `componentDidMount` to accomplish the same thing?